src/datasources/locking-api/entities/fingerprint-unsealed-data.entity.ts

```diff
@@ -29,7 +29,13 @@ export const FingerprintIpInfoSchema = z.object({
 export type FingerprintIpInfo = z.infer<typeof FingerprintIpInfoSchema>;
 
 export const FingerprintVpnSchema = z.object({
-  data: z.object({ result: z.boolean() }).nullish().default(null),
+  data: z
+    .object({
+      result: z.boolean(),
+      confidence: z.enum(['low', 'medium', 'high']),
```

Comment:
Nit: we should add an `unknown` value that we fallback to as [we do elsewhere](https://github.com/safe-global/safe-client-gateway/blob/main/src/domain/swaps/entities/order.entity.ts#L62) to ensure we cover unknowns. We should also add test coverage for this.